ts/src/editor.ts

```diff
@@ -215,10 +214,10 @@ function onPaste(elem) {
 }
 
 function caretToEnd() {
-    var r = document.createRange();
+    let r = document.createRange();
```

Comment:
You may already be aware of this, but const would be more appropriate in cases like this:  
https://mathiasbynens.be/notes/es6-const

Reply:
Thanks, done